pkgdb2/api/extras.py

```diff
@@ -158,18 +160,25 @@ def _vcs_acls_cache(out_format='text', eol=False, collection=None):
         eol=eol,
         collection=collection,
         oformat=out_format,
-        skip_pp=APP.config.get('PKGS_NOT_PROVENPACKAGER', None))
+        skip_pp=APP.config.get('PKGS_NOT_PROVENPACKAGER', None),
+        namespace=namespace)
     output = []
     if out_format == 'json':
-        output = {'packageAcls': packages,
-                  'title': 'Fedora Package Database -- VCS ACLs'}
+        output = packages
+        if 'rpms' in output:
```

Comment:
It was for backward compatibility but I'm fine with breaking it. It's an ``extra`` API endpoint anyway and we haven't promise to keep it stable.